app/views/curation_concerns/base/_chf_show_page.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="work-show-page-thumb">
```

Comment:
I'm not sure either! Have to review the code and re-figure it out and comment or alter appropriately, thanks!